### PR TITLE
Implement API route to return a specific Skill

### PR DESCRIPTION
**Pull request by nfallah**
This pull request closes issue #14. It allows users to query for specific skills via the 'id' query parameter; values outside of the valid range will return a generic error.

### CODE REVIEW

**Comment by nfallah** [comment]
@Pradyuman7

File: app.py [hunk 1]
```diff
@@ -123,7 +123,17 @@ def skill():
...
-        return jsonify({})
+        try:
+            skill_id = request.args.get('id')
+            if skill_id is not None:
+                skill_id = int(skill_id)
+                if 0 <= skill_id < len(data['skill']):
+                    return jsonify(data['skill'][skill_id]), 200
+                raise ValueError
+            else:
+                return jsonify({})
```
Comment (Pradyuman7) [inline app.py:134]: Do you think we should return 404 here as well?

File: app.py [hunk 2]
```diff
@@ -123,7 +123,17 @@ def skill():
...
+        except:
+            return jsonify({'error': 'Invalid request'}), 404
```
Comment (Pradyuman7) [inline app.py:136]: Maybe 400 here?

Comment (nfallah) [reply app.py]: I have added a commit that implements these proposed changes.

**Review by Pradyuman7 — APPROVED** [review]
LGTM 💯

**Pull request merged**